src/pages/gen2/build-a-backend/auth/override-cognito/index.mdx

```diff
@@ -40,9 +93,11 @@ const backend = defineBackend({
 // extract L1 CfnUserPool resources
 const { cfnUserPool } = backend.resources.auth.resources.cfnResources;
 // use CDK's `addPropertyOverride` to modify properties directly
-cfnUserPool.addPropertyOverride('Policies.PasswordPolicy.MinimumLength', 32);
+cfnUserPool.addPropertyOverride("Schema", [
+  {
+    Name: "publicName",
```

Comment:
can we show a custom or dev attribute example? that's one thing that's not available in `defineAuth`'s `userAttributes` prop

Reply:
does this overwrite the existing schema (with `email`)?

Reply:
Overriding `Schema` will add custom attributes. When I updated the `Schema` in this example, it didn't override the existing email updates I did. If there is an easier way of updating custom attributes let me know.